dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -79,6 +79,7 @@ def show
     @current_user = current_user && User.includes(:teachers).where(id: current_user.id).first
     authorize! :read, ScriptLevel
     @script = ScriptLevelsController.get_script(request)
+    @autoplay_enabled = current_user&.sections_as_student&.find_by(script_id: @script.id)&.autoplay_enabled
```

Comment:
Sorry to be a pain, but the term "autoplay" is already used in our system to refer to automatically playing the video on a level. please update this term to be more specific, e.g. `tts_autoplay_enabled` / `ttsAutoplayEnabled`.